saleor/graphql/order/bulk_mutations/orders.py

```diff
@@ -40,5 +42,7 @@ def bulk_action(cls, queryset, user, restock):
                 events.fulfillment_restocked_items_event(
                     order=order, user=user, fulfillment=order
                 )
-
+            manager = get_extensions_manager()
+            manager.order_cancelled(order)
+            manager.order_updated(order)
```

Comment:
Why running two webhooks here? Isn't "cancelled" enough? I'm wondering whether we shouldn't have one webhook like "order/updated" for all orders' state changes to make things simpler. I think Github does something like that e.g. for issues.

Reply:
Shopify + Magento have separate webhooks for that.  If you want to receive only cancel events you will subscribe to `order_cancelled`. 
Another service can be subscribed to receive all updates - `order_update`.
Those two services are assigned to separate events and both of them will recieve payload for different events.